src/Microsoft.OData.Core/UriParser/Aggregation/ApplyClause.cs

```diff
@@ -51,6 +53,13 @@ public ApplyClause(IList<TransformationNode> transformations)
 
                     break;
                 }
+                else if (transformations[i].Kind == TransformationNodeKind.Compute)
+                {
+                    if (lastAggregateExpressions == null)
+                    {
+                        lastComputeExpressions = (transformations[i] as ComputeTransformationNode).ComputeClause.ComputedItems;

```

Comment:
> lastComputeExpressions = (transformations[i] as ComputeTransformationNode).ComputeClause.ComputedItems; [](start = 23, length = 104)

In line 41 there is a break after lastAggregateExprssions is set. Do you want to do the same thing here?

Reply:
I see below that probably no.

---
In reply to: [173582650](https://github.com/kosinsky/odata.net/pull/9#discussion_r173582650) [](ancestors = 173582650)